### ISSUE TITLE
Soft LJ potentials for actin simulations

### ISSUE THREAD

**Issue by ezingrebe** [lhilbert/EnhancerShoeBox#9]
The data folder ~/Projects/ACTIN_SHOEBOX contains numerous subdirectories, each of them described by the suffix referring to the LJ soft potential, eg. box15-weak, box12-veryweak, box11-strong etc.

The main data folder contains only one file with the potentials: LJsoft_RSQ.table

Questions to be cleared out:

- how are the "weak", "veryweak", "medium", "strong" etc. potentials defined in the simulations? 

The simulation script: run_single_shoebox.py takes only one file, LJsoft_RSQ.table, as argument, which allows to assume that this file should already contain the desired potential values, and **only** those values - not a broad spectrum of possible potentials to choose between. However, the output data folder, ACTIN_SHOEBOX, contains only one LJsoft_RSQ.table; and this is the same file as for GENE_CLUSTER_SHOEBOX simulation folder. Hence, how were the simulation run in order to distinguish between weak, strong, medium etc. conditions?

- What does the "strong", "weak", "medium" etc. potential exactly mean? Concrete values (and in SI units!) are needed!

-Has it already been found, which potentials do reflect the experimental data best?

**Comment by lhilbert** [comment]
Could you already verify that running with the three options for the actin potentials actually results in different outcomes?

The reason I am asking, maybe this was intended as a feature, but has simply not been fully implemented...in that case we would be looking for an implementation that is simply not there. There were several development directions, and not all of them were fully finished.

**Comment by lhilbert** [comment]
Additional point: this is only relevant when simulating also actin, otherwise should not take effect

**Comment by ezingrebe** [comment]
Yes, the outcomes are very different. Just the comparison of eg. gene_stats.txt for strong, medium and veryweak potentials show significant differences in calculated values.

For the simulations without actin: the script run_single_shoebox.py calls LJsoft_RSQ.table file, but the values of alpha and lambda are set (and fixed) already in the script. So apparently the feature of different potentials is not implemented here.

**Comment by ezingrebe** [comment]
The README.md file contains the following comment for actin simulations: "The setting 'weak' seems to give the best results"
It is not explained, what "best" means (most stable simulations? lowest/no atomic overlap? agreement with experimental outcome?), but this is already some hint.
